contracts/Bridge.sol

```diff
@@ -71,6 +72,20 @@ contract Bridge is RelayerRole {
         emit QuorumChanged(newQuorum);
     }
 
+    /**
+        @notice Pause the contract
+    */
+    function pause() external whenNotPaused onlyAdmin {
```

Comment:
Ok, now that I see it implemented, being a simple logic, I think it would be better to implement `Pausable` ourselves instead of using openzepelin. Could you please implement it in shared structs as a component in order to avoid the code duplication for overrides? Sorry for the initial suggestion :)